src/transformers/generation/stopping_criteria.py

```diff
@@ -467,6 +467,18 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor, **kwa
         return is_done
 
 
+class ConfidenceCriteria(StoppingCriteria):
```

Comment:
Missing: docstring (most importantly, explaining what it should do and that is related to the method proposed in your paper)